aspnetcore/fundamentals/servers/kestrel/endpoints.md

```diff
@@ -199,7 +200,24 @@ In the following *appsettings.json* example:
 }
 ```
 
-An alternative to using `Path` and `Password` for any certificate node is to specify the certificate using certificate store fields. For example, the `Certificates` > `Default` certificate can be specified as:
+Schema notes:
+
+* Endpoints names are [case-insensitive](xref:fundamentals/configuration#configuration-keys-and-values). For example, `HTTPS` and `Https` are valid.
+* The `Url` parameter is required for each endpoint. The format for this parameter is the same as the top-level `Urls` configuration parameter except that it's limited to a single value.
+* These endpoints replace those defined in the top-level `Urls` configuration rather than adding to them. Endpoints defined in code via `Listen` are cumulative with the endpoints defined in the configuration section.
+* The `Certificate` section is optional. If the `Certificate` section isn't specified, the defaults defined in **Certificates** > **Default** are used. If no defaults are available then the development certificate is used. If there are no defaults and the development certificate isn't present then the server throws an exception and fails to start.
```

Comment:
```suggestion
* The `Certificate` section is optional. If the `Certificate` section isn't specified, the defaults defined in `Certificates:Default` are used. If no defaults are available, the development certificate is used. If there are no defaults and the development certificate isn't present, the server throws an exception and fails to start.
```

Reply:
`**Certificates** > **Default**` would have been localized.

Reply:
I'm going to update this one place, but `**Certificates** > **Default**` is used in many other places in this doc.